idaes/gas_solid_contactors/flowsheets/ss_MB_methane_combustion.py

```diff
@@ -112,14 +112,26 @@ def main():
             'Fe3O4': blk.solid_inlet.mass_frac_comp[0, 'Fe3O4'].value,
             'Al2O3': blk.solid_inlet.mass_frac_comp[0, 'Al2O3'].value}}
 
+    print()
```

Comment:
Should use the IDAES logger instead of print statmenets.

Reply:
This is in the example flowsheet not in the unit model.